service/src/main/resources/config/vumc/sdd_refresh0323/ui/top_level.json

```diff
@@ -170,6 +236,172 @@
         "start_date_group_by_count"
       ]
     },
+    {
+      "type": "classification",
+      "id": "tanagra-phewas",
+      "title": "PheWAS",
+      "conceptSet": true,
+      "category": "Source Codes",
+      "columns": [
+        { "key": "name", "width": "100%", "title": "Concept name" },
+        { "key": "code", "width": 120, "title": "Code" },
+        { "key": "label", "width": 120, "title": "Label" },
```

Comment:
What is the difference between `label`, `code`, and `name`? Do we need all three?

Reply:
e.g. name='Surgery', code='0234', label='0234 - Surgery'

I think we need `name` and `code` to display in the list/hierarchy views. I'm not sure we need `label`, which is just a concatenation of `name` and `code`. I added `label` because that's what the AoU-CB shows in some places, and I wasn't sure how to do the concatenation in the UI code, so instead I pointed it at a new attribute.

Reply:
Seems to me we probably don't want a column for it then but we can look and adjust it later.